pkgs/by-name/li/libusbp/package.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, udev
+, cmake
+, pkg-config
+}:
+
+stdenv.mkDerivation(finalAttrs: {
+  pname = "libusbp";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
+    owner = "pololu";
+    repo = "libusbp";
+    rev = "refs/tags/${finalAttrs.version}";
+    sha256 = "sha256-60xpJ97GlqEcy2+pxGNGPfWDnbIFGoPXJijaErOBXQs=";
+  };
+
+  outputs = [ "out" "dev" ];
+
+  nativeBuildInputs = [ cmake pkg-config ];
```

Personally, I prefer this formatting:

```suggestion
  nativeBuildInputs = [
    cmake
    pkg-config
  ];
```

As it makes future diffs smaller and easier to review.